examples/next/pages/ui/components/liveness/with-credential-provider/index.page.tsx

```diff
@@ -0,0 +1,35 @@
+import dynamic from 'next/dynamic';
+import React from 'react';
+
+import { Amplify } from 'aws-amplify';
+import { AwsCredentialIdentityProvider } from '@aws-sdk/types';
```

Comment:
Should we just be using the "copied" type here for consistency?

Reply:
We currently don't export any of our types including the credentials so we would need to update it to export that. Do we want to change that?

Reply:
Would there be value in exposing that type for package consumers? Guess a better question is what are we telling users to use?

Reply:
Currently in the android/ios docs there is little to no recommendation on what you should use since we are treating this as kind of an escape hatch - https://ui.docs.amplify.aws/android/connected-components/liveness

Originally our idea was to suggest using some of the sdk provided credential providers but Francisco warned against that since on the JS side they have found them to have issues

Reply:
A potential concern with informing users to utilize the SDK types but internally using our own types is the possibility of drift between the types of the user installed SDK version and our "copied" types

Reply:
sync'd offline, decided to export the types for now